drm/ResourceManager.cpp

```diff
@@ -105,10 +135,25 @@ void ResourceManager::Init() {
     // is SRI-IOV + dGPU, use virtio-gpu(card2) for display
     if (node_num == 3) {
       std::ostringstream path;
-      path << path_pattern << 2;
+      path << path_pattern << 1;
       auto dev = DrmDevice::CreateInstance(path.str(), this);
```

Comment:
we will call CreateInstance for both 1 and 2, while only emplace_back for 2, is it expected behavior?